courseraresearchexports/exports/utils.py

```diff
@@ -116,88 +94,3 @@ def lookup_partner_short_name_by_id(partner_id):
     response = requests.get(requests.compat.urljoin(PARTNER_API, partner_id))
```

Comment:
do we need to do similar error handling as above? and is there a way to abstract this pattern?